docs/ntp.md

```diff
@@ -26,6 +26,15 @@ ntp_servers:
   - "3.your-ntp-server.org iburst"
 ```
 
+## Setting the TimeZone
+
+The timezone can also be set by the `ntp_timezone` , eg: "Etc/UTC","Asia/Shanghai". If not set, the timezone will not change.
+
+```ShellSession
+ntp_enabled: true
+ntp_timezone: Etc/UTC
```

Comment:
We should add some commit, or install tzdata package.

Reply:
Thanks @liupeng0518 , the tzdata has been added.